product_variant_inactive/tests/test_button.py

```diff
@@ -58,3 +58,11 @@ def help_create_product(self, active=True):
             {"active": active, "name": "test_product"}
         )
         return product
+
+    def test_create_variant_do_not_reactive(self):
+        # Ensure that generating againt the variant do not change
```

Comment:
suggest: Ensure that re-generating variants does not change the "active" state of the existing variant

Reply:
fixed